.doc_gen/metadata/ssm_metadata.yaml

```diff
@@ -100,3 +219,20 @@ ssm_UpdateOpsItem:
                 - ssm.Java2.resolve_ops.main
   services:
     ssm: {UpdateOpsItem}                 
+ssm_Scenario:
+  title: Get started with the &SYS; SDK
+  title_abbrev: Get started with &SYS; 
+  synopsis: work with &SYS; maintaince windows, documents, and Opsitems.
+  category: Scenarios
+  languages:
+    Java:
+      versions:
+        - sdk_version: 2
+          github: javav2/example_code/ssm
+          sdkguide:
+          excerpts:
+            - description:
+              snippet_tags:
+                - ssm.java2.scenario.main
+  services:
+    ssm: {createOpsItem, createMaintenanceWindow, createDocument, sendCommand, commandInvocations, deleteMaintenanceWindow, updateOpsItem}
```

Comment:
The casing for Actions must match the REST API: https://docs.aws.amazon.com/systems-manager/latest/APIReference/Welcome.html, which is typically PascalCase, not camelCase.

Reply:
Fixed